TwilioVerifyDemo/TwilioVerifyDemo/TwilioVerifyDemo.entitlements

```diff
@@ -4,5 +4,13 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.twilio.TwilioVerifyDemo</string>
+	</array>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)com.twilio.TwilioVerifyDemo</string>
```

Comment:
What is the value of $(AppIdentifierPrefix)?

Reply:
> What is the best practice for this? Are other libraries using a helper to get the access group?

It vary from project to project, but basically it depends on project's configuration, so for a simple configuration using this kind of helpers will make things easier, without writing down the TeamID of the Company in the file. But if the app has too many variants in they group/id & keychain/id will require something different or more robust.

> What is the value of $(AppIdentifierPrefix)?

This a symbolic reference that Apple creates in the entitlement and resolves it during compilation/archiving time. The above helper allows to read that value once it reach the keychain.